examples/api/test/material/floating_action_button_location/standard_fab_location.0_test.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/floating_action_button_location/standard_fab_location.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('FloatingActionButton', (WidgetTester tester) async {
```

Comment:
Consider replacing the description with something more explicit.

Reply:
Oops sorry about that. I changed it in [test: Make the test description more explicit](https://github.com/flutter/flutter/pull/149225/commits/cd5433c9b079237dc300b11bd493c13b891fd158). Tell me if you have a better description in mind